server/controllers/routes/activities.js

```diff
@@ -0,0 +1,11 @@
+const getClassActivities = require('../../database/quieres/getClassActivities');
+
+module.exports = (req, res, next) => {
+  const { subjectId, classId } = req.params;
```

Comment:
validation - throw an error if subjectId or classId is not set. Consider throwing an error also for cases when the subject or class is not found in the database (ie. you get zero rows as a result).

Reply:
If the subjectid or the classid are not set then express won't match the route

Reply:
which status code will be sent in URL params not valid (400 or 422)

Reply:
right I see now 👍